src/volue/mesh/_base_session.py

```diff
@@ -805,6 +805,46 @@ def update_rating_curve_versions(
             :doc:`mesh_rating_curve`
         """
 
+    @abc.abstractmethod
+    def run_simulation(
+        self,
+        model: str,
+        case: str,
+        start_time: datetime,
+        end_time: datetime,
+        resolution: Timeseries.Resolution,
+        scenario: int,
+        return_datasets: bool,
+    ) -> Union[typing.Iterator[None], typing.AsyncIterator[None]]:
+        """Run a hydro simulation using HydSim on the Mesh server.
+
+        Args:
+            model: The name of the Mesh model in which the simulation case exists.
+            case: The names of the case group and simulation case on the form
+                'CaseGroup/CaseName'.
+            start_time: The (inclusive) start of the simulation interval.
+            end_time: The (exclusive) end of the simulation interval.
+            resolution: The resolution of the simulation. Can be left undefined
+                to use the default resolution defined in the simulation case.
+                **Unimplemented.**
+            scenario: The scenario(s) to run. **Unimplemented.**
+            return_datasets: **Unimplemented.**
+
+        Returns:
+            An iterator of `None`. In future versions this iterator will yield
+            log messages, datasets, and potentially more. The simulation is
+            done when the iterator is exhausted.
+
+            Exhausting the iterator without an exception does not guarantee
+            that the simulation completed successfully. To determine that
+            you must analyse the simulation's result time series and the
```

Comment:
Regarding "analyse". Are we using American or British English in Python's SDK documentation? Personally I used American English, although it was not explicitly agreed on.

Reply:
Changed to analyze for consistency.

Reply:
Will discuss internally.